src/libs/select.c

```diff
@@ -255,11 +372,29 @@ void init(struct dt_lib_module_t *self)
   lua_pushcclosure(L, dt_lua_type_member_common, 1);
   dt_lua_type_register_const_type(L, my_type, "register_selection");
 
+  lua_pushlightuserdata(L, self);
+  lua_pushcclosure(L, lua_destroy_selection,1);
+  dt_lua_gtk_wrap(L);
+  lua_pushcclosure(L, dt_lua_type_member_common, 1);
+  dt_lua_type_register_const_type(L, my_type, "destroy_selection");
+
+  lua_pushlightuserdata(L, self);
+  lua_pushcclosure(L, lua_set_selection_sensitive,1);
+  dt_lua_gtk_wrap(L);
+  lua_pushcclosure(L, dt_lua_type_member_common, 1);
+  dt_lua_type_register_const_type(L, my_type, "set_sensitive");
+
   dt_lua_module_entry_push(L,"lib",self->plugin_name);
   lua_getuservalue(L,-1);
   lua_newtable(L);
   lua_setfield(L,-2,"callbacks");
   lua_pop(L,2);
+
+  dt_lua_module_entry_push(L,"lib",self->plugin_name);
```

Comment:
note that style is to have one space after a coma for readability. there is multiple instances.

Reply:
fixed